docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,282 @@
+
+### Create the org MSP needed to add an org to a channel
+
+The organizations in a Fabric network don't exist in a physical sense the way nodes do. Rather, they exist as a structure of folders and certificates on the configuration of a channel. These certificates identify the relevant root CA, intermediate CA (if one was used), TLS CA, and at least one admin identity. As you recall from the membership topic and the registration and enrollment steps above, these folders and certificates are returned by the Fabric CA client when enrolling an admin identity, which is why the act of enrolling an admin and the act of "creating an organization" are closely related.
+
+Here is a sample of the folder structure you will need if you want to add an organization to a channel (the structure might vary slightly depending on the method you use to add an organization to a channel, but whatever the method, these are the files and folders you will need):
+
+```
+<location of msp>/msp
+├── config.yaml
+├── cacerts
+│   └── cacert.crt
+├── intermediatecerts
+|   └── cacert.crt
+├── tlscacerts
+│   └── tlsca.<org-domain>.pem
+└── tlsintermediatecerts
+    └── tlsca.<org-domain>.pem
+├── users
+    └── <admin public cert>.pem
+```
+
+Where the folders and certificates are:
+
+* `cacerts`: the root cert of the organization CA where the identity of the admin was registered and enrolled.
+* `intermediatecerts`: the root cert of an intermediate CA, if one was used.
+* `tlscacerts`: the root cert of the TLS CA that has issued certificates to the nodes associated with this organization.
+* `tlsintermediatecerts`: the root cert of the intermediate TLS CA, if one was used.
+* `users`: this folder must contain the **public** certificate of at least one admin of this organization. Do not add a private key here. If you have registered and enrolled multiple admins, you will only need to include the public certificate from those enrollments here. You will still need to point to the relevant private key to sign requests, however.
+
+Note that while the certificates themselves can be named anything you want, you should not change the name of the folders themselves, as Fabric expects to consume folders with certain names.
+
+### Creating the local MSP of a node
+
+While the MSP of an organization serves as the representation of the organization on a channel configuration, the local MSP of a node is a logical collection of parameters that is used, along with other parameters, as part of the creation of a node.
+
+As has been noted above, nodes must be bootstrapped both with enrollment certificates (the public/private key pair that identifies a node) and the TLS certificates that encrypt the communication layer between nodes. This "bootstrapping" happens by listing the location of these certificates in the relevant YAML configuration file that is referenced when creating the node. This means that **the local MSP of a node must be created before the node itself can be created**.
```

Comment:
This "bootstrapping" happens by listing the location of these certificates in the relevant YAML configuration file that is referenced when creating the node. 

I thought the bootstrapping happened by pointing to the location of the msp folder  in the relevant YAML file?

Reply:
If you look at core.yaml, the TLS certs section asks for the location of the individual certs, not a TLS MSP (or whatever it would be called): 

https://github.com/hyperledger/fabric/blob/master/sampleconfig/core.yaml#L252

So I used more general language as a result. But this is probably a worthy thing to make explicit.